spread.yaml

```diff
@@ -75,6 +75,9 @@ backends:
             - fedora-27-64:
                 image: fedora-27-64-selinux-permissive
                 workers: 4
+            - fedora-28-64:
+                image: fedora-28-64-permissive
```

Comment:
Nitpick: the 27 image is called "selinux-permissve" but this one is just "permissive". I'd be nice if this was consistent.